caravel/assets/javascripts/SqlLab/components/SqlEditorLeftBar.jsx

```diff
@@ -157,14 +181,15 @@ class SqlEditorLeftBar extends React.PureComponent {
           />
         </div>
         <div className="m-t-5">
-          <Select
+          <Select.Async
```

Comment:
Does that mean that we do a roundtrip at every keystroke? Is the substring in the cachekey? Seems like it would be more laggy for the per-schema use case